waddrmgr/db.go

```diff
@@ -38,6 +39,8 @@ var (
 	latestMgrVersion uint32 = LatestMgrVersion
 )
 
+type obtainFunc func() ([]byte, error)
```

Comment:
This should be exported since it's used in the Confg struct. It'll need a comment too once exported.